### PR TITLE
mobile: support Metamask QRs

### PR DESCRIPTION
**Pull request by dcposch**

### CODE REVIEW

File: packages/daimo-api/src/db/db.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { Address, Hex, getAddress } from "viem";
 /** Credentials come from env.PGURL, defaults to localhost & no auth. */
 const dbConfig: ClientConfig = {
   connectionString: process.env.PGURL,
-  connectionTimeoutMillis: 5000,
+  connectionTimeoutMillis: 10000,
```
Comment (nalinbhardwaj) [inline packages/daimo-api/src/db/db.ts:8]: drop?

File: apps/daimo-mobile/src/view/screen/QRScreen.tsx [hunk 2]
```diff
@@ -155,6 +155,14 @@ export function QRCodeBox({
 // Parse QR codes from Daimo or other wallets
 // Works around potential deep linking / AASA bugs by using direct links only
 function parseQRData(data: string) {
+  console.log(`[SCAN] parsing QR data '${data}'`);
+  if (data.startsWith("ethereum:")) {
+    data = data.slice("ethereum:".length).split("@")[0];
```
Comment (nalinbhardwaj) [inline apps/daimo-mobile/src/view/screen/QRScreen.tsx:160]: this is a new one, curious where is this from / what's the format? 👀

**Review by nalinbhardwaj — APPROVED** [review]

File: apps/daimo-mobile/src/view/screen/QRScreen.tsx (hunk 2 above)
Comment (dcposch) [reply apps/daimo-mobile/src/view/screen/QRScreen.tsx:160]: ya it's wack

see https://github.com/status-im/status-mobile/issues/19986#issue-2290795999

**Comment by dcposch** [comment]
merged to `master`, too

**Pull request merged**